apparmor.d/groups/lxqt/pcmanfm-qt

```diff
@@ -0,0 +1,110 @@
+  include <abstractions/fontconfig-cache-read>
+  include <abstractions/graphics>
+  include <abstractions/lxqt>
+  include <abstractions/nameservice-strict>
+  include <abstractions/recent-documents-write>
+  include <abstractions/thumbnails-cache-write>
+
+  signal (send) set=(term, kill),
+  signal (receive) set=(term, kill) peer=lxqt-session,
+
+  network netlink raw,
+
+  #aa:dbus own bus=session name=org.pcmanfm.PCManFM
+  #aa:exec kioworker
+
+  @{exec_path} mr,
+
+  @{lib}/menu-cache/menu-cached rPx,
+  @{lib}exec/menu-cache/menu-cache-gen rix,
+
+  owner @{user_cache_dirs}/pcmanfm-qt/**  r,
+  owner @{user_config_dirs}/pcmanfm-qt/lxqt/ r,
+  owner @{user_config_dirs}/pcmanfm-qt/lxqt/recent-files.conf.lock rwk,
+  owner @{user_config_dirs}/pcmanfm-qt/lxqt/desktop-items-0.conf.@{rand6} l -> @{user_config_dirs}/pcmanfm-qt/lxqt/#@{int},
+  owner @{user_config_dirs}/pcmanfm-qt/lxqt/dir-settings.conf~ l -> @{user_config_dirs}/pcmanfm-qt/lxqt/dir-settings.conf,
+  owner @{user_config_dirs}/pcmanfm-qt/lxqt/desktop-items-eDP-@{int}.conf.lock rwk,
+  owner @{user_config_dirs}/pcmanfm-qt/lxqt/settings.conf.lock rwk,
+  owner @{user_config_dirs}/pcmanfm-qt/lxqt/settings.conf.@{rand6} l -> @{user_config_dirs}/pcmanfm-qt/lxqt/#@{int},
+  owner @{user_config_dirs}/pcmanfm-qt/lxqt/desktop-items-0.conf.lock rwk,
+  owner @{user_config_dirs}/pcmanfm-qt/lxqt/desktop-items-eDP-@{int}.conf l -> @{user_config_dirs}/pcmanfm-qt/lxqt/#@{int},
+  owner @{user_config_dirs}/pcmanfm-qt/lxqt/recent-files.conf.@{rand6} l -> @{user_config_dirs}/pcmanfm-qt/lxqt/#@{int},
+  owner @{user_config_dirs}/pcmanfm-qt/lxqt/settings.conf.lock.* rwk,
+
+  @{sys}/bus/ r,
+  @{sys}/class/ r,
+  @{sys}/devices/system/node/ r,
+  @{sys}/devices/system/node/node@{int}/meminfo r,
+  @{sys}/fs/cgroup/{,**} r,
+  
+  owner @{PROC}/@{pid}/mountinfo r,
+  owner @{PROC}/@{pid}/mounts r,
+  owner @{PROC}/@{pid}/fd/ r,
+  owner @{PROC}/@{pid}/cgroup r,
+
+  # To read/write files in the system. The read permission is granted for all files, the write
+  # permission only for the owner. Also, dirs like /dev/, /efi/, /proc/, /sys/ are not included in
+  # the list.
+        /         r,
+        /boot/    r,
+        /boot/**  r,
+  owner /boot/**  rw,
+        /etc/     r,
+        /etc/**   r,
+  owner /etc/**   rw,
+        /home/    r,
+        /home/**  r,
+        /home/**  rw,
+        /lost+found/ r,
+        /lost+found/** r,
+  owner /lost+found/** rw,
+        @{MOUNTS}/   r,
+        @{MOUNTS}/** r,
+  owner @{MOUNTS}/** rw,
+        /opt/     r,
+        /opt/**   r,
+  owner /opt/**   rw,
+        /root/    r,
+        /root/**  r,
+  owner /root/**  rw,
+        @{run}/   r,
+        @{run}/** r,
+  owner @{run}/** rw,
+        /srv/     r,
+        /srv/**   r,
+  owner /srv/**   rw,
+        /tmp/     r,
+        /tmp/**   r,
+  owner /tmp/**   rw,
+        /usr/     r,
+        /usr/**   r,
+  owner /usr/**   rw,
+        /var/     r,
+        /var/**   r,
+  owner /var/**   rw,
```

Comment:
No, the policy in this project is to only allow user files. This can be changed by the end user, see https://apparmor.pujol.io/configuration/#file-browsers.

Here you need:
https://github.com/roddhjav/apparmor.d/blob/eda29668ae75d8b42412f35e3737230c6a626c09/apparmor.d/groups/gnome/nautilus#L84-L105

End these rules need to be moved before `owner @{user_cache_dirs}...`